### DISCUSSION TITLE
Alpha 185

### DISCUSSION THREAD

**Discussion by Eugeny** [Eugeny/tabby#7379] [Release Notes]
# Important changes

`xterm-webgl` is now the default terminal frontend - if you encounter issues with it, you can change it back to `xterm` in Settings -> Terminal. Keep in mind that the setting only applies to newly opened tabs.

# New features

- 512d2da: added the missing select-all hotkey - fixes #3081, fixes #3141
- 48afeb9: added a new right click behaviour matching windows terminal - fixed #7083
- 1171330: added minimum contrast ratio setting
- 7ed7ec3: fixed #7322 - made whitespace trimming on paste optional
- 0ea2adf: fixed #3862 - option to close the window when closing all tabs

# Changes

- 12f971d: fixed #7030 - directional pane navigation
- 96b3a40: fixed #7004 - terminal not focusing after exiting search
- 973dd55: fixed missing en-GB translation - #7199
- e1c07e7: set xterm-webgl as default frontend
- aa439cc: make ctrl-f select the search query
- a29a552: fixed linkifier plugin not working with multiple providers
- d3982f5: fixed #7288 - resize PTY when zooming in/out
- 270f273: fixed #7028 - only trim starting whitespace in pasted text if it's a single line

<hr /><em>This discussion was created from the release <a href='https://github.com/Eugeny/tabby/releases/tag/v1.0.185'>Alpha 185</a>.</em>

**Comment by berserker5000** [comment]
Hi, when the Tabby will be available for MacOs 13 Ventura? I can't leave without this great app :(

**Reply by Eugeny** [reply]
It works on Ventura.

**Reply by berserker5000** [reply]
Thanks, I probably had something with my mac, restarting of laptop and re-installing tabby helped.
